tailwind.config.ts

```diff
@@ -13,6 +13,15 @@ const config: Config = {
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
+      width: {
+        '1150px':'1150px',
+        '1000px':'1000px',
+        '500px':'500px',
+      },
+      height: {
+        '830px':'830px',
+      }
```

Comment:
🤔 why do we need this here?

Reply:
If we need an element with 1150px, how are we supposed to do it?